gradle.properties

```
@@ -15,5 +15,5 @@
 #
 
 group=org.sonatype.gradle.plugins
-version=3.0.0-SNAPSHOT
+version=3.1.0-SNAPSHOT
```

There is no change in the plugin code, if the intention is to test this releasing a new version then it's better just to increment the patch number:
```suggestion
version=3.0.1-SNAPSHOT
```